packages/testing/src/chai-retry-plugin/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import Chai from 'chai';
+import { timeout as addTimeoutToPromise, sleep } from 'promise-assist';
+import { chaiMethodsThatHandleFunction } from './constants';
+
+import type { RetryAndAssertArguments } from './types';
+
+const { expect } = Chai;
+
+const performRetries = async ({ functionToRetry, options, assertionStack }: RetryAndAssertArguments) => {
+    const { retries, delay } = options;
+    let retriesCount = 0;
+
+    while (retriesCount < retries) {
+        try {
+            retriesCount++;
+            /**
+             * This is needed to handle cases when function that user passes to `expect`
```

Comment:
This comment is still too verbose in my opinion. I'd suggest: 
```
If assertion chain includes method such as `increase` (and some others), function passed to the `expect` will be called by Chai itself.
```
or something in that vein

Reply:
@yurii-ve fixed

Reply:
@yurii-ve done